packages/replay/README.md

```diff
@@ -112,11 +110,11 @@ The `replay.(min.)js` bundle will be removed in v8 of the JS SDKs.
 
 ```html
 <script
-  src="https://browser.sentry-cdn.com/7.31.1/bundle.min.js"
```

Comment:
😬 - how do we keep this up to date?

Reply:
I guess we don't 😬 maybe it's time to remove this from these docs and instead just link to docs? We can do this in a follow up.

Reply:
Sounds good to me